src/pages/workspaces/List.js

```diff
@@ -291,15 +293,9 @@ export const WorkspaceList = _.flow(
     return h(Fragment, [
       h(TopBar, { title: 'Workspaces' }, [h(WsSearch, { onChange: v => this.setState({ filter: v }) })]),
       h(PageBox, { style: { position: 'relative' } }, [
-        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'flex-end', marginBottom: '1rem' } }, [
+        div({ style: { display: 'flex', alignItems: 'center', marginBottom: '1rem' } }, [
           div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Workspaces']),
-          div({ style: { marginLeft: 'auto' } }, [
-            h(LabeledCheckbox, {
-              checked: includePublic === true,
-              onChange: v => this.setState({ includePublic: v })
-            }, ' Show public workspaces')
-          ]),
-          div({ style: { marginLeft: '1rem', flex: '0 0 300px' } }, [
+          div({ style: { marginLeft: 'auto', flex: '0 0 300px' } }, [
```

Comment:
Could we update this to add a `flexGrow` spacer before this instead? It's the pattern we tend to use elsewhere.